ZenPacks/zenoss/Import4/model.py

```diff
@@ -143,6 +143,12 @@ def zenpack(self):
 
         log.info("zenpacks restored:%s", codeString[ExitCode.SUCCESS])
 
+        # copy the license files for ucspm if exist
```

Comment:
ok.. I'll move it to the database portion.
As for the bash, functionally, it is simpler to handle file/processes with bash scripts.
Just an artificial separation of arena.